### PR TITLE
Add "Getting Started" lab to introduce Terraform concepts

### PR DESCRIPTION
**Pull request by zefdelgadillo**
The 01-getting-started lab aims to introduce Terraform concepts for users/students who may have not used the tool before. It also walks through the general lab flow to get users familiar with running Terraform commands.

### CODE REVIEW

**Comment by zefdelgadillo** [comment]
@bharathkkb as another reviewer

File: 06-Cloud-Function/README.md [hunk 1]
```diff
@@ -1,15 +1,28 @@
 
 # Cloud Foundation Toolkit Lab - 05-Cloud-Function
+<<<<<<< HEAD:05-Cloud-Function/README.md
```
Comment (bharathkkb) [inline 06-Cloud-Function/README.md]: Conflict markers in this file

**Review by bharathkkb — CHANGES_REQUESTED** [review]

File: 06-Cloud-Function/README.md (hunk 1 above)
Comment (zefdelgadillo) [reply 06-Cloud-Function/README.md]: Thanks for the catch

**Review by bharathkkb — APPROVED** [review]

**Comment by zefdelgadillo** [comment]
@marine675 can you also take a look when you have some time?

File: 01-Getting-Started/README.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+## Prerequisites
+* Google Cloud Platform project with valid billing account
+* [Terraform](https://learn.hashicorp.com/terraform/getting-started/install.html)
+* [Google Cloud SDK](https://cloud.google.com/sdk/install)
+
+## Setup
+This lab helps you initiate Terraform to work with Google Cloud Platform.
+
+At the end of the exercise, you will have 2 GCS buckets (**Logs** and **Remote State**) with 4 lines of output value.
```
Comment (marine675) [inline 01-Getting-Started/README.md]: Where are the `output` blocks?

Comment (marine675) [reply 01-Getting-Started/README.md]: Only seeing 1 GCS bucket for remote state

**Review by marine675 — CHANGES_REQUESTED** [review]

File: 01-Getting-Started/README.md (hunk 2 above)
Comment (zefdelgadillo) [reply 01-Getting-Started/README.md]: I opted for just a single GCS bucket for state to isolate and must have missed this part of the documentation. I'll clean it up.

Output block is a good idea for instruction also as part of getting-started.

Comment (zefdelgadillo) [reply 01-Getting-Started/README.md]: cebadab should take care of this

**Review by marine675 — APPROVED** [review]
LGTM

**Pull request merged**